pkgs/default.nix

```diff
@@ -22,24 +23,58 @@ rec {
   active-modules = import ./active-modules {
     inherit pkgs;
     inherit self;
+    inherit all-modules;
   };
 
   upgrade-maps = import ./upgrade-maps {
     inherit pkgs;
   };
 
-  bundle-locked = pkgs.callPackage ./bundle-locked { inherit revstring; };
-
   bundle-image = pkgs.callPackage ./bundle-image {
     inherit bundle-locked revstring;
     inherit active-modules upgrade-maps;
   };
 
   bundle-image-tarball = pkgs.callPackage ./bundle-image-tarball { inherit bundle-image revstring; };
 
-  bundle-squashfs = pkgs.callPackage ./bundle-squashfs {
-    inherit bundle-locked revstring;
-    inherit active-modules upgrade-maps;
+  bundle-locked-fn = { modulesLocks }: pkgs.callPackage ./bundle-locked {
```

Comment:
This and the other -fn break `nix flake show`, we either need to move them up into the let block at the top of this file or make a new lib output for the flake and then reference them via that.